README.md

```diff
@@ -34,14 +31,12 @@ npm install blob-to-buffer
 ### usage
 
 ```js
-var toBuffer = require('blob-to-buffer')
+const toBuffer = require('blob-to-buffer')
 
 // Get a Blob somehow...
-var blob = new Blob([ new Uint8Array([1, 2, 3]) ], { type: 'application/octet-binary' })
-
-toBuffer(blob, function (err, buffer) {
-  if (err) throw err
+const blob = new Blob([ new Uint8Array([1, 2, 3]) ])
 
+toBuffer(blob).then(buffer => {
```

Comment:
Can you switch this example to use `await`?

Reply:
sure, i only used `.then()` as it was more cross platform compatible and many ppl just copy/paste code without using babel